sdgym/benchmark.py

```diff
@@ -504,6 +507,14 @@ def benchmark_single_table(synthesizers=DEFAULT_SYNTHESIZERS, custom_synthesizer
         pandas.DataFrame:
             A table containing one row per synthesizer + dataset + metric.
     """
+    if run_on_ec2:
+        if output_filepath is not None:
```

Comment:
we should also probably print something to the user saying we are going to launch an instance on their account